packages/wix-ui-icons-common/src/on-stage/system/metadata.ts

```diff
@@ -36,6 +36,13 @@ const tpaIconsMetadata: Array<IconMetadata> = [
     tags: [],
     sizes: { "32":"EmailLarge" },
   },
+  {
+    title: "Indeterminate",
+    category: "General",
+    description: "",
+    tags: [],
+    sizes: { "16": "Indeterminate"}
```

Comment:
16px means that it is the `XSmall` size.

```suggestion
    sizes: { "16": "IndeterminateXSmall""}
```